WalkingChallenge/Challenge/Challenge.swift

```diff
@@ -31,4 +31,161 @@ import UIKit
 import Foundation
 
 class Challenge: UIViewController {
+  private let lblTitle: UILabel = UILabel(typography: .headerTitle)
+  private let btnSettings: UIButton = UIButton(type: .system)
+  private let tblCards: UITableView = UITableView(frame: .zero)
+
+  private var arrCards: [JourneyCardMiles] = []
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    self.navigationController?.navigationBar.barTintColor = #colorLiteral(red: 0.9647058823, green: 0.9725490196, blue: 0.9803921568, alpha: 1.0000000000)
+    self.navigationController?.navigationBar.setValue(true, forKey: "hidesShadow")
+
+    layout()
+  }
+
+  private func layout() {
+    view.backgroundColor = #colorLiteral(red: 0.9647058823, green: 0.9725490196, blue: 0.9803921568, alpha: 1.0000000000)
+
+    lblTitle.text = Strings.Challenge.title
+
+    lblTitle.textColor = .black
+
+    view.addSubview(lblTitle)
+    lblTitle.snp.makeConstraints {
+      $0.top.equalTo(topLayoutGuide.snp.bottom)
+      $0.left.right.equalToSuperview().inset(Style.Padding.p16)
+    }
+
+    btnSettings.setImage(UIImage(named: "GearIcon"), for: .normal)
+
+    view.addSubview(btnSettings)
+    btnSettings.snp.makeConstraints {
+      $0.top.equalTo(topLayoutGuide.snp.bottom)
+      $0.right.right.equalToSuperview().inset(Style.Padding.p16)
```

Comment:
The button here will have the same constraints as the title label which seems wrong.